### DISCUSSION TITLE
sidewalk where cycling is allowed

### DISCUSSION THREAD

**Discussion by BER319** [streetcomplete/StreetComplete#5289] [Feedback]
I miss the following option for sidewalks with bicycle access.

https://wiki.openstreetmap.org/wiki/Key:sidewalk:both:bicycle

**Comment by westnordost** [comment]
Hayupp, see #4913

**Reply by westnordost** [reply]
In a nutshell, streetcomplete does not use this tag because if it is set, it is not clear whether it is set because there is a sign like displayed on the wiki page, or because it is generally allowed due to legislation. But this is important for verifiability. Hence, the option can only be added if it is additionally tagged explicitly that it is signed - see #4913.

The reason I did not implement #4913 yet is that I haven't really gotten the feeling that this is really wanted by the community. Till now, there are only like 2 thumbs-up on this ticket 🤷‍♂️.

**Reply by BER319** [reply]
thank you for the answer. since there is already a discussion can be closed here.
